.circleci/config.yml

```diff
@@ -1,15 +1,38 @@
 version: 2.0
 jobs:
-  build:
+  test:
     docker:
       - image: membraneframeworklabs/docker_membrane:latest
-    environment:
-      MIX_ENV: test
-    working_directory: '~/app'
+        environment:
```

Comment:
Is the change of the indentation intentional?

Reply:
tbh I've just copied it from `membrane_template_plugin`

Reply:
Ok